searx/engines/xpath.py

```diff
@@ -166,6 +171,9 @@ def request(query, params):
         'safe_search': safe_search,
     }
 
+    params['cookies'] = cookies
+    params['headers'] = headers
```

Comment:
It should be
```python
    params['cookies'].update(cookies)
    params['headers'].update(headers)
```

because the User Agent is already set.

---

https://github.com/searxng/searxng/blob/c474616642890af97ce79700baff7dd47b2406b1/searx/search/processors/online.py#L57-L61